terragrunt-run-all/action.yml

```diff
@@ -0,0 +1,34 @@
+name: Terragrunt Run All
+description: Terragrunt run-all
+inputs:
+  folder:
+    description: The target folder
+    required: true
+  iam-role:
+    description: The iam role to assume
+    required: true
+  bin-name:
```

Comment:
why are the path to and name of `terragrunt` parameterized? is it unsafe to assume terragrunt will be configured on `PATH` before the action is called?

Reply:
I parameterized them to avoid assumptions that could not be overwritten. The default value for the terragrunt location is /usr/local/bin (and default name of binary being terragrunt). When used in conjuction with the terragrunt-setup action, the binary will be in the expected location so as not to require overwriting the defaults. However if terragrunt was downloaded or installed by another means, then the terragrunt location can be explicitly specified. Looking at it again, it does seem overly complicated to parameterize both values, and I could consolidate them into a single input, for example "terragrunt-bin-path".